internal/broker/broker.go

```diff
@@ -183,3 +192,15 @@ func TLSConfigBroker(b Mqconf) *tls.Config {
 	}
 	return cfg
 }
+
+// // One would typically keep a channel of publishings, a sequence number, and a
+// // set of unacknowledged sequence numbers and loop until the publishing channel
+// // is closed.
+func confirmOne(confirms <-chan amqp.Confirmation) error {
```

Comment:
What is this? Why do we have a queue of confirmations?

Reply:
When writing to a durable exchange we want confirmation that the message has been persisted.